src/borg/testsuite/helpers.py

```diff
@@ -683,13 +682,13 @@ def test_get_cache_dir_compat(monkeypatch):
     monkeypatch.delenv("BORG_BASE_DIR", raising=False)
     monkeypatch.delenv("XDG_CACHE_HOME", raising=False)
     if not is_darwin and not is_win32:
-        # Please try again on darwin and win32 and update test accordingly
+        # Please try on darwin and win32 and update comment accordingly
```

Comment:
This needs to be done still - both macOS and win32 could be check via github CI and some temporary commit that fails, so one sees the values and can write some similar comments as seen in the config compat test.